internal/run/target.go

```diff
@@ -420,3 +440,37 @@ func (w *Workflow) CancelGeneration() error {
 
 	return fmt.Errorf("Generation is not cancellable")
 }
+
+func writeSdkChangelogToDisk(ctx context.Context, changelogContent string, lang string, logger logging.Logger) error {
+	err := storePullRequestMetadata(ctx, fmt.Sprintf("SDK_CHANGELOG_%s", lang), changelogContent)
+	if err != nil {
+		log.From(ctx).Warnf("error computing changes: %s", err.Error())
+	}
+
+	// Add commit message
+	err = storePullRequestMetadata(ctx, fmt.Sprintf("%s_commit_message", lang), changelogContent)
```

Comment:
This will override itself for monorepos where there are multiple targets for the same language